SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/Spatial2WorkerMap.cpp

```diff
@@ -23,6 +25,7 @@ void USpatial2WorkerMap::CreateCustomContentForMap()
 	AddActorToLevel<ARegisterAutoDestroyActorsTestPart1>(CurrentLevel, FTransform::Identity);
 	AddActorToLevel<ARegisterAutoDestroyActorsTestPart2>(CurrentLevel, FTransform::Identity);
 	AddActorToLevel<ARelevancyTest>(CurrentLevel, FTransform::Identity);
+	AddActorToLevel<ASpatialTestMultiServerUnrealComponents>(CurrentLevel, FTransform(FVector(250, -250, 0)));
```

Comment:
```suggestion
	// Test is placed on one half of the map to make it authoritative on Server 1
	AddActorToLevel<ASpatialTestMultiServerUnrealComponents>(CurrentLevel, FTransform(FVector(250, -250, 0)));
```

Reply:
You mean 'Actor is placed'?

Reply:
I guess either one could be used, but the actor is an instance of ASpatialFunctionalTest, so I would call it a test normally.
Or even better, 'Test actor'.

Reply:
'Test actor' reads well.